### DISCUSSION TITLE
Inside-out Alpha Wraps

### DISCUSSION THREAD

**Discussion by NahomKidane** [CGAL/cgal#8190] [Q&A] [answered]
Hello, 

Could we use the 3D alpha wrapping to carve from within to obtain an inner volume? They did mention that capability on the original paper (https://inria.hal.science/hal-03688637). I am not sure if this was added to CGAL.

**Comment by MaelRL** [comment, marked as answer]
See [this example](https://github.com/CGAL/cgal/blob/master/Alpha_wrap_3/examples/Alpha_wrap_3/wrap_from_cavity.cpp).

**Reply by NahomKidane** [reply]
Great Thanks!
